frontend/src/features/admin-form/responses/ResponsesPage/storage/useDecryptionWorkers.ts

```diff
@@ -181,7 +190,24 @@ const useDecryptionWorkers = ({
                       errorCount++
                       console.error('Error in getResponseInstance', e)
                     }
+
                     if (downloadAttachments && decryptResult.downloadBlob) {
+                      // Ensure attachments downloads are spaced out to avoid browser blocking downloads
+                      if (progress % ATTACHMENT_DOWNLOAD_CONVOY_SIZE === 0) {
```

Comment:
1. Since `timeSinceLastXAttachmentDownload = 0` on the initialization step
2. Then `elaspedSinceXDownloads = now - timeSinceLastXAttachmentDownload` will be set to a value that's higher than `1000`
3. Then `ATTACHMENT_DOWNLOAD_CONVOY_MINIMUM_SEPARATION_TIME - elaspedSinceXDownloads` will be a negative number
4. Then `Math.max(0, val)` will be evaluated as `0`
5. Then we will fail the wait condition

Thus, the first convoy doesn't have to wait.

Lmk if I've gotten one of the steps wrong 🙈

Reply:
Yeap, my bad. I guess the only extra "overhead" is the few extra instructions executed which is ok